### ISSUE TITLE
The effect of depth task only using unsupervised pre-trained model

### ISSUE THREAD

**Issue by zhangshuoneu** [cvg/depthsplat#9]
Hi, wondeful work!
According to the paper, the network is surpervised using photometric error. And for specific depth estimation tasks, the depth data are used to fine-tune. I wonder if the effect of pre-trained models on deep tasks has been tested without applying depth supervised fine-tuning. I am curious about the effect of unsupervised training on deep tasks. It would be great to have your guidance!

**Comment by haofeixu** [comment]
Hi, the unsupervisedly pre-trained depth model produces overall reasonable depth predictions. However, the details might not be very pricise since they are trained with the photometric loss. After fine-tuning with ground-truth depth supervision, they can be very sharp, as we show in the project page.

**Comment by zhangshuoneu** [comment]
Thanks for your reply!
